kuttl-tests/suites/failure-recovery/node-failure/delete-node-for-pod.sh

```diff
@@ -6,6 +6,11 @@ NAMESPACE=$3
 
 NODE_NAME=`kubectl get pod ${POD_NAME} -n ${NAMESPACE} -o=custom-columns=NODE:.spec.nodeName --no-headers`
 
-echo "Delete node $NODE_NAME"
+echo "Cordon node $NODE_NAME"
+kubectl cordon ${NODE_NAME}
+
+echo "Deleting pod {$POD_NAME}"
+kubectl delete pod ${POD_NAME} -n ${NAMESPACE}
 
+echo "Delete node $NODE_NAME"
 kubectl delete node ${NODE_NAME}
```

Comment:
This seems to be the most working version right now - Cordon, delete the pod, then delete the node.

Draining doesn't work, just deleting the node leaves the C* process running sometime...